### PR TITLE
enhance(client): support login with browser

### PR DESCRIPTION
**Pull request by jialeicui**
## Description

https://github.com/star-whale/starwhale/assets/3217223/72d7d247-4560-4d59-9aaa-e11a2de19654

## Modules
- [x] UI
- [ ] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: console/src/pages/Auth/ClientLogin.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export default function ClientLogin() {
+    const [t] = useTranslation()
+    const { token } = useAuth()
+    const [success, setSuccess] = useState(false)
+    const [loading, setLoading] = useState(false)
+    const [errMsg, setErrMsg] = useState('')
+    useEffect(() => {
+        if (!token) {
+            return
```
Comment (goldenxinxing) [inline console/src/pages/Auth/ClientLogin.tsx:16]: Is it possible to login in broswer when the broswer without token?

File: client/starwhale/core/instance/view.py [hunk 2]
```diff
@@ -44,6 +48,68 @@ def login(self, instance: str, alias: str, **kw: str) -> None:
...
+        async def on_startup() -> None:
+            nonlocal instance
+            url = f"{instance}/auth/client"
+            import webbrowser
+
+            webbrowser.open(url)
+
+        @app.get("/login")
+        async def _login(token: str) -> dict:
+            print(f"get token {token}")
```
Comment (goldenxinxing) [inline client/starwhale/core/instance/view.py:80]: How about use console to logging

Comment (jialeicui) [reply client/starwhale/core/instance/view.py:80]: `console.info` did not work, I don't know why :rofl: 
I'll try to find out the reason

File: console/src/pages/Auth/ClientLogin.tsx (hunk 1 above)
Comment (jialeicui) [reply console/src/pages/Auth/ClientLogin.tsx:16]: It relies on the user being logged in, and after the console supports logging in and jumping to the pre-login page, the experience will be smoother.

Comment (goldenxinxing) [reply console/src/pages/Auth/ClientLogin.tsx:16]: ok, i see

File: client/starwhale/core/instance/view.py (hunk 2 above)
Comment (jialeicui) [reply client/starwhale/core/instance/view.py]: I forgot to set the log lv using `-vvv`  :face_with_spiral_eyes: 
removed this line, and change the success echo.

**Review by goldenxinxing — APPROVED** [review]
LGTM

**Pull request merged**